### DISCUSSION TITLE
I'm not able to set the shortcut with cmd+opt+ctrl + left arrow to move left. (⌘⌥⌃←)

### DISCUSSION THREAD

**Discussion by gwthm-in** [rxhanson/Rectangle#1497] [General]
I was using shift it and hammer-spoon which do not work correctly on my machine. So I moved to Rectangle and it works like a charm to move to the next screen which wasn't with Shiftit and Hammerspoon. 

I'm trying to set the shortcuts compatible with Hammerspoon, so I tried to set 

To move to Left Half I tried to set **(cmd+ctrl+opt+left arrow) ⌘⌥⌃←**, but it doesn't record. Is there a way to achieve it?

![image](https://user-images.githubusercontent.com/8710113/220905640-f11cf9d5-a318-4f04-81b9-50b8daf81e5d.png)

macOS version: Ventura 13.1 (22C65)
Rectangle version: 0.66 (72)

**Comment by rxhanson** [comment]
First try checking the box in the settings tab: "Remove shortcut recorder safeguards".

If you still can't record it, then it's likely being consumed by another app and isn't making it to Rectangle. 

I have plans to replace the shortcut recorder in an upcoming release, it's just a matter of getting a block of time I can commit to it. At that time, I'll be able to make some changes to better handle these types of scenarios.

**Comment by henkethebenke** [comment]
Same issue for me. I have checked the box "Allow all keyboard shortcuts" (roughly translated from Swedish) and it still won't let me bind ctrl-opt-cmd-left and right, however up and down works. Version 0.66 and I'm using Ventura 13.2.1.

**Comment by lucasfeliciano** [comment]
Same issue here!
I'm coming from Moom, and I was setting up the same layouts that I had there
I manged to set most of them but the `cmd+ctrl+opt+s` `⌘⌥⌃s` doesn't go through :(

![image](https://user-images.githubusercontent.com/968014/226342632-344e8751-1116-46e0-93ff-2fe37b1f7c2f.png)

**Comment by savetheclocktower** [comment]
I was all ready to say that this issue was happening for me as well, but it turned out that ⌘⌥⌃← and ⌘⌥⌃→ were already claimed by _different_ shortcuts in Rectangle. I don't think this is the case for the original bug reporter (if that first screenshot is accurate) but it might be useful for others who encounter this bug.

Once I cleared the shortcuts for “Next Display” and “Previous Display,” I was able to assign ⌘⌥⌃← and ⌘⌥⌃→ to “Left Half” and “Right Half,” respectively.

I think it might still be considered a bug that the shortcut recorder appears not to be able to listen properly for a shortcut that's already assigned. I'm sure it's complicated under the hood, but it seems like…

> If you still can't record it, then it's likely being consumed by another app and isn't making it to Rectangle.

…turned out to be true — only the shortcuts were being consumed by Rectangle themselves and still weren't making it to the shortcut recorder. Maybe a workaround would be to have the keyboard shortcuts screen indicate somehow when Rectangle is reacting to a shortcut — if the “Next Display” shortcut had flashed when I pressed ⌘⌥⌃→, I'd have more quickly realized what was going on.

(I was able to understand that Rectangle itself was responding to the shortcut by using [Shortcut Detective](https://www.irradiatedsoftware.com/labs/) — made by the same folks that made SizeUp, the software I've installed Rectangle to replace.)

**Comment by step21** [comment]
I have a similar issue. At first I couldn't bind ⌥⌘↑ to put the window into the top half. Then after restarting or sth it just was there and worked. But now after some time it stopped working. When I try to rebind it, it doesn't work, but if I cancel rebinding it is still there. I couldn't find any system shortcuts or other apps that use it so far. Shortcut detective doesn't seem to catch it either. It is not a hardware thing, as in other combinations/apps the keys on the keyboard all work. And "allow all shortcuts" is also ticked.

**Reply by rxhanson** [reply]
This still points to the shortcut being captured elsewhere, although I still don't have a good way to get to the bottom of it. Have you tried quitting every single running 3rd party app, trying the shortcut after each one to narrow it down?

**Reply by step21** [reply]
Thanks. I'll try that, but so far haven't found any app that would re-enable it by closing it.
